tests/main_test.py

```diff
@@ -104,6 +104,14 @@ def test_package_info_all_info():
     }
 
 
+def test_package_info_wheel_with_local_version():
```

Comment:
Instead of a new test, I think you can just add a case to `test_guess_name_version_from_filename` above

Reply:
@asottile - I considered that, but that test function doesn't cover the change - guess_name_version_from_filename() is called from inside create(), but only after the regex gets checked.

Reply:
ahh ok, let's go for a more directed test then, how about something like this?

```python
    pkg = main.Package.create(filename='f-1.0+local-py3-none-any.whl')
    assert pkg.version == '1.0+local'
```

Reply:
@asottile - done